src/coveralls.erl

```diff
@@ -62,6 +63,9 @@
 
 %%=============================================================================
 %% API functions
+% @doc Initialization plugin for rebar3
+-spec init(rebar_state:t()) -> {ok, rebar_state:t()}.
+init(State) -> rebar3_coveralls:init(State).
```

Comment:
Would it not be possible to call the plugin rebar3_coveralls, as the module is named?